README.md

```diff
@@ -80,7 +80,9 @@ Easy Scheduler uses a lot of excellent open source projects, such as google guav
 It is because of the shoulders of these open source projects that the birth of the Easy Scheduler is possible. We are very grateful for all the open source software used! We also hope that we will not only be the beneficiaries of open source, but also be open source contributors, so we decided to contribute to easy scheduling and promised long-term updates. We also hope that partners who have the same passion and conviction for open source will join in and contribute to open source!
 
 ### Get Help
-The fastest way to get response from our developers is to submit issues,  or add our wechat : 510570367
+1. Submit an issue
+1. Mail list: dev@dolphinscheduler.apache.org. Mail to dev-subscribe@dolphinscheduler.apache.org, follow the reply to subscribe the mail list.
+1. QQ Group: 510570367 for Mandarin(CN) discussion.
```

Comment:
I think QQ Group is not accurate. This 510570367 is the wechat number of core developer, he will help add into wechat groups.

Reply:
Updated.